Added Q&A section to dedicated resources (#4897)
umbraco/UmbracoDocs: umbraco-cloud/set-up/project-settings/dedicated-resources.md

_Dedicated Resources_ is a feature on Umbraco Cloud that gives you the option to move your project to a dedicated server. You can choose between a number of _dedicated options_ depending on the amount of resources you will need for your project.

> In this article, you can read about how to move your Umbraco Cloud project to dedicated resources. As well what you need to be aware of before you do so.

Reviewer: ```suggestion
In this article, you can read about how to move your Umbraco Cloud project to dedicated resources. You can also find information about what you need to be aware of before doing so.
```

* By hitting "Downgrade to shared" and confirming your choice, you will be redirected to the project page where you will be notified when the move back to a shared resource has been completed.
* Your Cloud project is now back on a shared resource.

## Frequently asked questions

Wondering what happens when you move your environment? 
Below you can find a list of the most frequently asked questions and the answers.

* Will it move all the environments to the dedicated server?
  * Yes, all environments will be moved to the dedicated server.
* How will the resources be split betweeen the environments?
  * All environments on the project moved to dedicated will share the ressources of the dedicated server.
* Will the customer be able to work on the project during the move?

>   * It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the website will not respond as usual

Reviewer 2: >It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the website will not respond as usual.

I would change this to Backoffice instead of website.

You can elaborate if you think that makes sense:
When its a move to dedicated within the same tier only the website resources are moved.
If its a move between Plans and to Dedicated, ie. from Starter to Standard Dedicated then the databases are also moved to a higher tier.
Because the website is moved and restarts and hostname switches happen everyone should refrain from working in the backoffice, because it might be switched over while someone is saving/publishing content.

Author: Makes sense with changing it to backoffice 👍

For now, I think it is fine to not go into too much detail about what happens behind the scenes.

We can always add it down the line if it is something people using cloud starts to ask more into it :)

* By hitting "Downgrade to shared" and confirming your choice, you will be redirected to the project page where you will be notified when the move back to a shared resource has been completed.
* Your Cloud project is now back on a shared resource.

## Frequently asked questions

Wondering what happens when you move your environment to a dedicated server? 

> Below you can find a list of the most frequently asked questions and the answers.

Reviewer: ```suggestion
Below you can find a list of the most frequently asked questions including answers.
```

1. **Will it move all the environments to the dedicated server?**
     - Yes, all environments will be moved to the dedicated server.
2. **How will the resources be split between the environments?**
     - All environments on the project moved to dedicated will share the resources of the dedicated server.
3. **Will the customer be able to work on the project during the move?**

>      - It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the backoffice will not respond as usual

Reviewer: ```suggestion
     - It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the backoffice will not respond as usual.
```

> 
> 1. Will it move all the environments to the dedicated server?
>   * Yes, all environments will be moved to the dedicated server.
> 2. How will the resources be split betweeen the environments?
>   * All environments on the project moved to dedicated will share the ressources of the dedicated server.
> 3. Will the customer be able to work on the project during the move?
>   * It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the backoffice will not respond as usual
> 4. Will the environments be moved at the same time or one by one?
>   * Yes, all environments will be moved simultaneously. 
> 5. Will the live environment be unavailable while the Project is moved?
>   * There will always be an active live environment which continues to serve requests and be online during the move operation. When the moved live environment is ready and responding to requests, the hostnames will be switched to point to the moved environment.

Author: ```suggestion

1. Will it move all the environments to the dedicated server?
  - Yes, all environments will be moved to the dedicated server.
2. How will the resources be split between the environments?
  - All environments on the project moved to dedicated will share the resources of the dedicated server.
3. Will the customer be able to work on the project during the move?
  - It will not be possible to work on the project while it is being moved to the dedicated server. The move takes a couple of minutes, and during that time the backoffice will not respond as usual
4. Will the environments be moved at the same time or one by one?
  - Yes, all environments will be moved simultaneously. 
5. Will the live environment be unavailable while the Project is moved?
  - There will always be an active live environment that continues to serve requests and be online during the move operation. When the moved live environment is ready and responding to requests, the hostnames will be switched to point to the moved environment.
```
